Source/Fuse.Controls.CameraView/Android/CameraImpl.java

```
@@ -253,6 +257,164 @@ public void setFlashMode(String flashMode) {
+
+                        if (focusModes.contains(Camera.Parameters.FOCUS_MODE_AUTO)) {
+                            parameters.setFocusMode(Camera.Parameters.FOCUS_MODE_AUTO);
+                        }
+                    }
+
+                    if (parameters.isAutoExposureLockSupported()) {
+
+                        parameters.setAutoExposureLock(false);
+                    }
+
+                    if (parameters.isAutoWhiteBalanceLockSupported()) {
+
+                        parameters.setAutoWhiteBalanceLock(false);
+                    }
+
+                    _camera.setParameters(parameters);
+                }
+
+                int focus_area_size = 300;
+                int left = clamp(Float.valueOf((float)(x / cameraWidth) * 2000 - 1000).intValue(), focus_area_size);
+                int top = clamp(Float.valueOf((float)(y / cameraHeight) * 2000 - 1000).intValue(), focus_area_size);
+
+
+                Display display = com.fuse.Activity.getRootActivity().getWindowManager().getDefaultDisplay();
+                Double tmp;
+                Rect rect;
+                switch(display.getRotation()) {
+                    case 0: //portrait
+                        rect = new Rect(left, top, left + focus_area_size, top + focus_area_size);
+                        break;
+                    case 3: //landscape right
+                        rect = new Rect(top, left, top + focus_area_size, left + focus_area_size);
+                        break;
+                    case 2: //portrait upsidedown
+                        rect = new Rect(top, left, top + focus_area_size, left + focus_area_size);
+                        break;
+                    case 1: //landscape left
+                        rect = new Rect(left, top, left + focus_area_size, top + focus_area_size);
+                        break;
+                    default:
+                        rect = new Rect(left, top, left + focus_area_size, top + focus_area_size);
```

Is this actually needed, or are you just playing safe here? If the latter, I'd just throw an exception or something instead. If not, I'd reuse the `case 0`-code to avoid repetition...